shareledger/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "shareledger",
+  "assets": [
+    {
+      "description": "The native token of Shareledger",
+      "denom_units": [
+        {
+          "denom": "nshr",
+          "exponent": 0,
+        },
+        {
+          "denom": "shr",
+          "exponent": 9,
```

Comment:
please remove the commas after the exponent numbers in both denoms, since a property doesn't immediately follow them.

Reply:
Ah sorry, forgot to remove those when I removed the empty attributes.

Reply:
Fixed @JeremyParish69